DPG/Tasks/qaEventTrackLite.cxx

```diff
@@ -267,6 +275,9 @@ struct qaEventTrackLite {
     }
 
     for (const auto& particle : particles) {
+      if (TMath::Abs(particle.dpgCollision().posZ() > 10.))
```

Comment:
Maybe we could use here the same configurable

Reply:
yes, of course, and also done via a Filter, but if you do not mind, I'll change it on Monday (or asap)

Reply:
Ok!

Reply:
I realized this was also wrong, the abs was on top of the bool selection, I fixed that in the last commit

Reply:
ah, right thanks! Sorry. Instead, for the filling of the histo, that's correct. It's not the same histo and must be filled both in the loop over tracks and in that over nonreco particles.